packages/generator-reactserver/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Huh.  Cool.  Didn't know about editorconfig.

But... why is this different from the rest of the `react-server` repo?